src/ripple/app/paths/Flow.h

```diff
@@ -58,6 +60,7 @@ flow (PaymentSandbox& view,
     bool defaultPaths,
     bool partialPayment,
     bool ownerPaysTransferFee,
+    bool offerCrossing,
     boost::optional<Quality> const& limitQuality,
     boost::optional<STAmount> const& sendMax,
```

Comment:
Or-able enums are an improvement over what's there. The downside is we can't require that all the flags be specified (at compile time). I think it's a good idea to force the caller to specify all these flags.

I know we frown on enum classes for boolean switches, but I think that's the best solution here. It's type safe, documents the parameters, and forces the caller to specify each parameter. I also don't want to re-litigate an issue that's already been decided. If there are objections I'd go with OR-able enums. @vinniefalco What do you think?